jacques/jacques.py

```diff
@@ -1,27 +1,31 @@
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..')) 
 import pandas as pd
 import numpy as np
 import tensorflow as tf
 import abc
 import math
 import random
-from featurize import featurize_data
+from jacques.featurize import featurize_data
```

Comment:
Attempting to reference from within same parent module?
```suggestion
from .featurize import featurize_data
```